compiler/AST/expr.cpp

```diff
@@ -1456,6 +1469,35 @@ CallExpr* ContextCallExpr::getRValueCall() {
   return toCallExpr(options.head);
 }
 
+void  ContextCallExpr::getCalls(CallExpr*& refCall,
+                                CallExpr*& valueCall,
+                                CallExpr*& constRefCall) {
+  refCall = NULL;
+  valueCall = NULL;
+  constRefCall = NULL;
+
+  if (options.length == 2) {
+    refCall = getRefCall();
+    CallExpr* rvalueCall = getRValueCall();
+    FnSymbol* fn = rvalueCall->isResolved();
+    INT_ASSERT(fn);
+    if (fn->retTag == RET_CONST_REF)
+      constRefCall = rvalueCall;
+    else
+      valueCall = rvalueCall;
+  } else if (options.length == 3) {
+    // Note: it would be nicer to check retTag to decide between
+    // ref / value versions. However, doing so is challenging because
+    // of the way that iterator functions no longer have the original
+    // retTag.
+    constRefCall = toCallExpr(options.get(1));
+    valueCall = toCallExpr(options.get(2));
+    refCall = toCallExpr(options.get(3));
+  } else {
+    INT_ASSERT(0 && "Bad ContextCallExpr options");
```

Comment:
Did you mean to use something else here, like INT_FATAL? Should the string be a comment instead?